src/messages/search/MessageFlagsPredicate.cpp

```diff
@@ -0,0 +1,42 @@
+#include "messages/search/MessageFlagsPredicate.hpp"
+
+namespace chatterino {
+
+MessageFlagsPredicate::MessageFlagsPredicate(const QStringList &flags)
+    : flags_()
+{
+    // Check if any comma-seperated values were passed and transform those
+    for (const auto &entry : flags)
+    {
+        for (const auto &flag : entry.split(',', QString::SkipEmptyParts))
+        {
+            if (flag == "deleted")
+            {
+                this->flags_.set(MessageFlag::Disabled);
+            }
+            else if (flag == "sub" || flag == "subscription")
```

Comment:
Another nitpick: Maybe `"subscribed"` could be added / replace `"subscription"`.

Reply:
Not 100% sure about this one :thinking: My "fear" is that it could easily be confused with [Filters](https://wiki.chatterino.com/Filters/) only showing the _messages_ of subscribed users.

Reply:
Yeah I think "subscription" fits "is" more, and "is:subscribed" could be confused with the author being subscribed (which might be another good option but not in this context). Either way though if you think it should be an option I can add it.

Reply:
> Not 100% sure about this one thinking My "fear" is that it could easily be confused with [Filters](https://wiki.chatterino.com/Filters/) only showing the _messages_ of subscribed users.

Yeah, my bad. I got confused first and thought that's what this flag does, but if it's just meant to show sub messages then it's fine as it is. Sorry for causing confusion ;p